packages/terra-list/src/MultiSelectList.jsx

```diff
@@ -138,7 +138,7 @@ class MultiSelectList extends React.Component {
 
     return (event) => {
       if (event.nativeEvent.keyCode === KEYCODES.ENTER) {
-        if (item.props.isSelectable !== false && this.shouldHandleSelection(index)) {
+        if (item.props.isSelectable && this.shouldHandleSelection(index)) {
           this.handleSelection(event, index);
         }
 
```

Comment:
Also we may need to break out the logic inside handleSelection. Set state isn't occuring synchronously, so we need to calculate the desired value for the onChange notifcation.

In single select, it will just be the new index.

Multi-select will need additional logic.